backend/api/manageUsersApi.js

```diff
@@ -20,6 +20,11 @@ const manageUsersApiFactory = (client) => {
 
   const createUser = (context, user) => post(context, '/prisonusers', user)
   const contextUserRoles = (context, username) => get(context, `/prisonusers/${username}/roles`)
+  const getAllEmailDomains = (context) => {
+    return client.get(context, `/email-domains`).then(processPageResponse(context))
+  }
```

Comment:
Your call to get all emaildomains is not a paged response. I may have missed this in the first review

Reply:
Nope, the page response is generic enough to handle non paged calls.